packages/runtime/test-runtime-utils/src/mocksForReconnection.ts

```diff
@@ -121,6 +121,21 @@ export class MockContainerRuntimeFactoryForReconnection extends MockContainerRun
 		return containerRuntime;
 	}
 
+	public getMinSeq(): number {
+		// We overwrite the min sequence number function, to simulate that
+		// the minimum sequence number will be incremented by the server while the
+		// client is not connected. By checking, whether a client is currently
+		// within the quorum we ignore it during the min sequence number computation
+		// it it is currently not connected.
+		let minSeq: number | undefined;
+		for (const [client, clientSeq] of this.minSeq) {
+			if (this.quorum.getMember(client) !== undefined) {
+				minSeq = minSeq === undefined ? clientSeq : Math.min(minSeq, clientSeq);
+			}
+		}
+		return minSeq ?? 0;
```

Comment:
If sequence numbers are never negative (which I think is true), this simplification should be valid:
```suggestion
		let minSeq = 0;
		for (const [client, clientSeq] of this.minSeq) {
			if (this.quorum.getMember(client) !== undefined) {
				minSeq = Math.min(minSeq, clientSeq);
			}
		}
		return minSeq;
```

Reply:
Unfortunately, it's not, as the only valid `minimumSequenceNumber` is the one existing in the `this.minSeq` map, and it can start with something greater than 0.